packages/terra-clinical-action-header/package.json

```diff
@@ -23,29 +23,29 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "devDependencies": {
-    "terra-toolkit": "^0.x"
+    "terra-toolkit": "^0.3.1"
   },
   "peerDependencies": {
-    "react": "15.4.2",
-    "react-dom": "15.4.2",
-    "terra-base": "^0.x",
-    "terra-button": "^0.x",
-    "terra-button-group": "^0.x",
-    "terra-icon": "0.x",
-    "terra-mixins": "^1.0.0",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.4.1",
+    "terra-button": "^0.4.1",
+    "terra-button-group": "^0.2.1",
+    "terra-icon": "^0.4.1",
+    "terra-mixins": "^1.1.1",
     "terra-clinical-header": "^0.x",
-    "terra-responsive-element": "^0.x"
+    "terra-responsive-element": "^0.3.1"
```

Comment:
I would just update

```
"react": "15.4.2",
"react-dom": "15.4.2",
```
to
```
"react": "^15.4.2",
"react-dom": "^15.4.2",
```

If you leave the dependencies from terra-core at `0.x` then you will get new changes as we release them.  We just need to be careful in terra-core about preventing/communicating non-passive changes.